src/utils.js

```diff
@@ -21,29 +21,34 @@ export function getServerUrlForBrowser (url) {
         return `${scheme}${window.location.hostname}${port}/ws`;
     }
 
-    if (url.startsWith('/')) {    // just path on current server
+    if (url.startsWith('/')) { // just path on current server
         return `${scheme}${window.location.hostname}${port}${url}`;
     }
 
     // assuming just domain + path
     return `${scheme}${url}`;
 }
 
-export function getWebSocket (url, protocols, ws, headers, requestOptions) {
-    const parsedUrl = isNode ? getServerUrlForNode(url) : getServerUrlForBrowser(url);
+
+export function getWebSocket (wampy = {}) {
```

Comment:
As it is not clear from the function definition what options it uses, I think we need at least to describe it in @JSDoc format: prop + format + optional? + desc
example: 
```js
/**
  * @param {Object} userInfo Information about the user.
  * @param {string} userInfo.name The name of the user.
  * @param {string} userInfo.email The email of the user.
  */
 function logIn(userInfo) {
        doLogIn(userInfo.name, userInfo.email);
 }
```